Source/WebCore/dom/Node.h

```diff
@@ -200,6 +201,8 @@ class Node : public EventTarget {
     WEBCORE_EXPORT const AtomString& lookupPrefix(const AtomString& namespaceURI) const;
     WEBCORE_EXPORT const AtomString& lookupNamespaceURI(const AtomString& prefix) const;
 
+    WEBCORE_EXPORT std::optional<std::variant<String, RefPtr<TrustedScript>>> textContentForBinding();
+    WEBCORE_EXPORT virtual ExceptionOr<void> setTextContentForBinding(const std::optional<std::variant<String, RefPtr<TrustedScript>>>&);
```

Comment:
Can we use three overloads here instead of `std::variant`? Seems like it would be higher performance and easier to read the code. I guess this depends on what the bindings generator makes.

Reply:
I don't think that works because the IDL definition uses a union which translates to a variant. I don't think WebKit's IDL generator supports the concept of overloading?

Reply:
I've looked into this and I can't find a way to use overloading for attributes, if these were functions it'd be easy but because they're attributes it seems you can only really have one getter and setter.